public_configs/nightly_tests/NixVM_meterpreter_nightly.json

```diff
@@ -0,0 +1,107 @@
+{
+	"HTTP_PORT":			5309,
+	"STARTING_LISTENER":		30000,
+	"CREDS_FILE":			"../JSON/creds.json",
+	"MSF_HOSTS":
+	[
+		{
+			"TYPE":			"VIRTUAL",
+			"METHOD":		"VM_TOOLS_UPLOAD",
+			"HYPERVISOR_CONFIG":	"../JSON/esxi_config.json",
+			"CPE":			"cpe:/a:rapid7:metasploit:::",
+			"MSF_PATH":		"/home/msfuser/rapid7/metasploit-framework",
+			"MSF_ARTIFACT_PATH":	"/home/msfuser/rapid7/test_artifacts"
+		}
+	],
+	"TARGETS":
+	[
+		{
+		    "CPE": "cpe:/o:centos:centos:6::x64", 
```

Comment:
This target format is incorrect should be 

```TARGETS":
	[
		{
		    "CPE": "cpe:/o:centos:centos:6::x64"
		},
		    "CPE": "cpe:/o:centos:centos:7::x64"
		},
		    "CPE": "cpe:/o:fedoraproject:fedora:25::x64"
		},
		    "CPE": "cpe:/o:fedoraproject:fedora:26::x64"
		},
		    "CPE": "cpe:/o:fedoraproject:fedora:27::x64"
		},
		    "CPE": "cpe:/o:fedoraproject:fedora:28::x64"
		},
		    "CPE": "cpe:/o:fedoraproject:fedora:29::x64"
		},
		    "CPE": "cpe:/o:canonical:ubuntu_linux:14.04::x64"
		}, 
		    "CPE": "cpe:/o:canonical:ubuntu_linux:16.04::x64"
		},
		    "CPE": "cpe:/o:canonical:ubuntu_linux:18.04::x64"
		}
	],
```

Reply:
I think you missed some opening braces?

Reply:
Also, that means we'll need to fix the others, but let's not do the others, here.